toqito/helper/bell_notation_conversions.py

```diff
@@ -0,0 +1,748 @@
+                    v_mat[a, ob - 1, x, y] = k_term + a_term
+                # Fill V[oa-1, b, x, y] for b < ob-1 (last row for Alice)
+                for b in range(ob - 1):
+                    b_term = cg_mat[0, bindex(b, y)] / ia if ia > 0 else 0
+                    v_mat[oa - 1, b, x, y] = k_term + b_term
+                # Fill V[oa-1, ob-1, x, y] (bottom-right corner)
+                v_mat[oa - 1, ob - 1, x, y] = k_term
+
+    else:  # behaviour == 1
+        for x in range(ia):
+            for y in range(ib):
+                # Calculate slices for CG matrix corresponding to non-last outcomes
+                # Need 1-based indices for slicing cg_mat
+                start_row_a = aindex(0, x)
+                end_row_a = aindex(oa - 2, x) + 1 if oa > 1 else start_row_a
+                slice_a = slice(start_row_a, end_row_a)
+
+                start_col_b = bindex(0, y)
+                end_col_b = bindex(ob - 2, y) + 1 if ob > 1 else start_col_b
+                slice_b = slice(start_col_b, end_col_b)
+
+                # Get corresponding submatrix or default to zeros/scalars if outputs=1
+                cg_sub_mat = cg_mat[slice_a, slice_b] if oa > 1 and ob > 1 else np.array([[]])
+                cg_a_marg = cg_mat[slice_a, 0] if oa > 1 else np.array([])
+                cg_b_marg = cg_mat[0, slice_b] if ob > 1 else np.array([])
+
+                # V(0..oa-2, 0..ob-2, x, y) = p(a,b|xy)
+                if oa > 1 and ob > 1:
+                    v_mat[0 : oa - 1, 0 : ob - 1, x, y] = cg_sub_mat
+
+                # V(0..oa-2, ob-1, x, y) = pA(a|x) - sum_{b'=0..ob-2} p(a,b'|xy)
+                if oa > 1:
+                    sum_b = np.sum(cg_sub_mat, axis=1) if ob > 1 else np.zeros(oa - 1)
+                    v_mat[0 : oa - 1, ob - 1, x, y] = cg_a_marg - sum_b
+
+                # V(oa-1, 0..ob-2, x, y) = pB(b|y) - sum_{a'=0..oa-2} p(a',b|xy)
+                if ob > 1:
+                    sum_a = np.sum(cg_sub_mat, axis=0) if oa > 1 else np.zeros(ob - 1)
+                    v_mat[oa - 1, 0 : ob - 1, x, y] = cg_b_marg - sum_a
+
+                # V(oa-1, ob-1, x, y) = 1 - sum pA(a|x) - sum pB(b|y) + sum p(ab|xy)
+                sum_a_marg = np.sum(cg_a_marg)
+                sum_b_marg = np.sum(cg_b_marg)
+                sum_ab_joint = np.sum(cg_sub_mat)
+                v_mat[oa - 1, ob - 1, x, y] = (
+                    cg_mat[0, 0]  # Should be 1 for behaviour
+                    - sum_a_marg
+                    - sum_b_marg
+                    + sum_ab_joint
+                )
+
+    return v_mat
+
+
+def fc_to_fp(fc_mat: np.ndarray, behaviour: bool = False) -> np.ndarray:
+    r"""Convert a Bell functional or behaviour from Full Correlator (FC) to Full Probability (FP) notation.
+
+    Assumes binary outcomes (oa=2, ob=2) corresponding to physical values +1 and -1.
+    The FP tensor indices `a, b = 0, 1` correspond to outcomes `+1, -1` respectively.
```

Comment:
```suggestion
    The FP tensor indices ``a, b = 0, 1`` correspond to outcomes ``+1, -1`` respectively.
```
Either use inline math mode or double `